x-pack/plugins/infra/public/components/asset_details/asset_details.stories.tsx

```diff
@@ -98,6 +107,16 @@ const stories: Meta<AssetDetailsProps> = {
       memoryFree: 34359738368,
     },
     overrides: {
+      overview: {
+        dataView: {
```

Comment:
I couldn't find a good way to add the types here as I wanted to add this just to mock the data view in order to load the overview tab